resources/hiveclients/java/hiveserver2-client/README.md

```diff
@@ -0,0 +1,26 @@
+#hiveserver2-client
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/82cd4d3adc4ea715c3e18fda9273a74016edba69